sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -580,6 +580,9 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     options?: OperationOptionsBase & SendManagementRequestOptions
   ): Promise<Long[]> {
     throwErrorIfConnectionClosed(this._context);
+    if (!messages.length) {
+      return [];
```

Comment:
Changelog?

Reply:
I have quite a few PRs out, am worried slightly about the merge conflicts in the changelog and the potential delays from waiting for builds to finish. Will update the changelog at one go before the release